File: PL020.cpp
#include <iostream>
using namespace std;

int gcdOfTwoNumbers(int a, int b)
{
    while (a != b)
    {
        if (a > b)
        {
            a -= b;
        }
        else
        {
            b -= a;
        }
    }

    return a;
}

int main()
{
    int a, b;
    cout << "Enter the number a : ";
    cin >> a;
    cout << "Enter the number b : ";
    cin >> b;
    // cout << endl;

    cout << "Gcd of two numbers " << a << " and " << b << " is : " << gcdOfTwoNumbers(a, b) << endl;

    return 0;
}

File: PL020.java
import java.util.Scanner;

public class PL020 {
    public static void main(String[] args) {
        int a, b;
        Scanner scanner = new Scanner(System.in);
        System.out.println("Enter the number a : ");
        a = scanner.nextInt();
        System.out.println("Enter the number b : ");
        b = scanner.nextInt();
        scanner.close();

        System.out.println("Gcd of the given number is " + gcdOfTwoNumbers(a, b));

    }

    public static int gcdOfTwoNumbers(int a, int b) {
        while (a != b) {
            if (a > b) {
                a -= b;
            } else {
                b -= a;
            }
        }

        return a;
    }
}
